index.js

```diff
@@ -557,6 +557,20 @@ exports.createFileSystem = function (volume, opts, cb) {
         else _.delayedCall(cb, S.err.NOSYS());
     }, (_n_ === '_nested_')); };
     
+    fs.createLabel = function (name, cb) {
+        fs.open(name, "a", (e,_fd) => {
+            if (e) {
+                throw '_open_error_'
```

Comment:
should be `return cb(e);`

Reply:
Is `return` required? I don't think callback has to return anything.

Reply:
And fixed